MediaBrowser.Model/Notifications/NotificationOption.cs

```diff
@@ -1,3 +1,4 @@
+#pragma warning disable CA1819 // Properties should not return arrays
```

Comment:
Why not fix it instead?

Reply:
Because that would probably require a larger rewrite of the code depending on where these are used.

 public string[] DisabledMonitorUsers { get; set; }
 public string[] SendToUsers { get; set; }